File: MyTestVueApp.Server/Entities/Artist.cs
namespace MyTestVueApp.Server.Entities
{
    public class Artist
    {
        public int Id { get; set; } //Public id
        public string SubId { get; set; } //SECRET ID
        public string Name { get; set; }
        public bool IsAdmin { get; set; } = false;
        public bool PrivateProfile { get; set; } = false;
        public DateTime CreationDate { get; set; }
        public string Email { get; set; } = "";
    }
}

File: mytestvueapp.client/src/entities/Artist.ts
export default class Artist {
  //required
  id: number;
  name: string;
  isAdmin: boolean;
  privateProfile: boolean;
  creationDate: string;
  email: string;

  constructor() {
    this.id = 0;
    this.name = "";
    this.isAdmin = false;
    this.privateProfile = false;
    this.creationDate = "";
    this.email = "";
  }
}
